bitrepository-client/src/main/java/org/bitrepository/commandline/eventhandler/PutFileEventHandler.java

```diff
@@ -54,15 +55,18 @@ public PutFileEventHandler(Settings settings, OutputHandler outputHandler, boole
     @Override
     public void handleComponentComplete(OperationEvent event) {
         if (!(event instanceof PutFileCompletePillarEvent)) {
-            output.warn("PutFileEventHandler received a component complete, which is not a "
-                    + PutFileCompletePillarEvent.class.getName());
+            output.warn("PutFileEventHandler received a component complete, which is not a " + PutFileCompletePillarEvent.class.getName());
```

Comment:
I put my max line # to 140 because it fits the screen better - to be honest we aren't running any clear code-guidelines but in my opinion we should sit down and clarify some, and then put them on **Bitrepository.org**. There already is a guideline page but it is outdated and for sure hasn't been used in the project for ages.